independent-projects/resteasy-reactive/pom.xml

```diff
@@ -314,7 +314,7 @@
             </dependency>
             <dependency>
                 <groupId>io.smallrye.stork</groupId>
-                <artifactId>smallrye-stork-api</artifactId>
+                <artifactId>stork-core</artifactId>
```

Comment:
Shouldn't it depend on the bom?

Reply:
Yeah, it would be better to import the BOM here too.

Reply:
I wasn't sure about it. It's an independent project. I can change to Bom.

Reply:
changed to bom import